### ISSUE TITLE
Qtractor 

### ISSUE THREAD

**Issue by Reaper10** [rncbc/qtractor#32]
qtractor could be a open source alterative to Adobe Audition you just need to add a Waveform Editor like this 
![1-037-037272-adobe-audition-1-5](https://cloud.githubusercontent.com/assets/8137561/16464382/8ef6a4c4-3dff-11e6-8af4-4c6da97b4cd5.jpg)
![adobe-audition-cs6](https://cloud.githubusercontent.com/assets/8137561/16464381/8ef68368-3dff-11e6-8321-7e921c62be35.png)
![adobe-audition-cs-5 5](https://cloud.githubusercontent.com/assets/8137561/16464380/8ef5d51c-3dff-11e6-8000-dfeb6d38ea7f.jpg)
https://helpx.adobe.com/audition/using/displaying-audio-waveform-editor.html
I don't know why I want to help being I can't use it.

**Comment by rncbc** [comment]
hell, yes, but we have audacity for that :)

**Comment by Reaper10** [comment]
I thought these would help with recording and cleaning up audio.

**Comment by alandmoore** [comment]
I would agree that reimplenting a full audio editor inside of qtractor would be a wasteful duplication of effort, however -- the workflow of using Audacity and Qtractor together has a lot of tedius overhead involved.  

Say I've got a 2-minute audio file, which I've sliced up into a number of smaller clips in a track.  I want to apply some kind of effect or destructive edit to just one of those clips.   

Right now I have to open Audacity, locate that 2-minute audio file, find the bit of it that I wanted to edit, make the edits, export the audio, and hope it didn't jack up the timing for all the other clips off that audio file.

It would be so nice to be able to right-click that one clip and have it open directly in an editor (audacity or another of my choosing), see only the selected portion (which qtractor could copy out of the original clip file) to edit; then, after editing and saving, have qtractor replace that clip with the new file (leaving the original audio file intact, so that the other clips aren't affected).

There are probably a lot of challenges to that, not the least of which is how audacity handles opening files, but if there's any way to move qtractor in a direction where making quick edits to audio clips is less of an ordeal, I think that would be a welcome improvement.

**Comment by Reaper10** [comment]
couldn't you just add Gnome Wave to qtractor ?

**Comment by alandmoore** [comment]
Not that I presume to speak for rncbc, but if an editor were to be integrated it would probably help if it was also written in C++ and used the QT toolkit.  GWC is written in C and uses Gnome libs.

**Comment by rncbc** [comment]
back in the days i was tempted by rezound (http://rezound.sf.net) back when its author announced he was working on moving to the Qt platform. unfortunately, development has stalled for almost a decade now. too bad. rezound was already a top notch wave editor on linux, way far and better then than audacity now, imho.
